packages/cms/README.md

```diff
@@ -466,6 +466,27 @@ However, be warned that this is not "true" javascript, merely string manipulatio
 /api?hierarchy=<parents[-1].value>
 ```
 
+### Accessing Member Slugs
+
+Member slugs only exist at the CMS level for vanity-plate URL routing (e.g., /profile/country/fra, where `fra` is the member slug). The underyling cube has no knowledge of these slugs, which can make creating links to these vanity URLs in the CMS difficult.
+
+If you want access to these slugs in your results set, you may add the `slugs` query parameter to your generator API. The CMS will intercept the payload and inject the slugs into the response.
+
+The slugs parameter requires two elements for a successful lookup:
+
+1) The CMS-level dimension on which the ID is considered unique (`Exporter`, `HS Product`, etc)
+2) An accessor for the key in the response payload to be used for lookup (`Country`, `HS4`, etc). Note: The CMS will automatically append the ` ID` to your accessor, changing `HS4` to `HS4 ID` for example.
+
+These parameters should be added to the generator API, using colons to separate the two required pieces:
+
+`&slugs=Exporter:Country,HS Product:HS4`
+
+If the pieces are the same, one parameter may be used:
+
+`&slugs=Product`
+
+**Warning** This slug parameter **will not work** if your result set has more than 10 elements. Remember to use `limit` to keep your generator calls tiny!
```

Comment:
Hmm, I understand why we would want to put in a limit, but 10 seems kind of... small. @jhmullen did you do any testing to see how much of a performance hit it is to do it on say, 100 rows? My gut says:
- this default number could/should be a little higher
- there should be an env var that allows users to change this limit

Reply:
@davelandry I'd rather take off the limit entirely than add a config. There is only one `db.search` call per colon-separated pair (I use findAll and make a hashmap). In my OEC testing, performance does not seem to degrade. I think it's best to remove the limit entirely, this optimization is more liable to cause confusion than help us in the speed department.